lib/demos/reference/motion_demo_fade_through_transition.dart

```diff
@@ -0,0 +1,207 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN fadeThroughTransitionDemo
+
+class FadeThroughTransitionDemo extends StatefulWidget {
+  const FadeThroughTransitionDemo();
+
+  @override
+  _FadeThroughTransitionDemoState createState() =>
+      _FadeThroughTransitionDemoState();
+}
+
+class _FadeThroughTransitionDemoState extends State<FadeThroughTransitionDemo> {
+  int pageIndex = 0;
+
+  List<Widget> pageList = [
```

Comment:
Make this final, private, and move the typing to the List declaration.
```suggestion
  final _pageList = <Widget>[
```